checker/raw_result.go

```diff
@@ -136,11 +136,28 @@ type VulnerabilitiesData struct {
 	Vulnerabilities []clients.Vulnerability
 }
 
+type SecurityPolicyInformationType string
+
+const (
+	// forms of security policy hints being evaluated.
+	SecurityPolicyInformationTypeEmail SecurityPolicyInformationType = "emailAddress"
+	SecurityPolicyInformationTypeLink  SecurityPolicyInformationType = "httpLink"
+	SecurityPolicyInformationTypeText  SecurityPolicyInformationType = "vuln & disclosure text"
+)
+
+type SecurityPolicyInformation struct {
+	InformationType  SecurityPolicyInformationType
+	InformationValue []string
```

Comment:
how will you keep track of lines each of the `InformationValue` were found?

Reply:
changed `InformationValue []string` to a new struct to track "telemetry" of matches/hits in the policy file aligning with the `SecurityPolicyInformationType` constants as in:

```
type SecurityPolicyValueType struct {
       Match      string   // Snippet of match
       LineNumber uint     // Line number in policy file of match
       Offset     uint     // Offset in the line of the match
}
```

as such, the new `SecurityPolicyInformation` type tracks this telemetry for each match/hit as in:

```
 type SecurityPolicyInformation struct {
       InformationType  SecurityPolicyInformationType
       InformationValue SecurityPolicyValueType
 }
```